Source/SLKTextViewController.h

```diff
@@ -122,6 +122,7 @@ NS_CLASS_AVAILABLE_IOS(7_0) @interface SLKTextViewController : UIViewController
 @property (nonatomic, readonly) UIButton *leftButton;
 @property (nonatomic, readonly) UIButton *rightButton;
 
+- (void)slk_updateViewConstraints;
```

Comment:
This needs to be public so we can rejigger their constraint logic when we need to